### PR TITLE
added more examples

### PR DESCRIPTION
**Pull request by brettmortensen**
Added some more workflow examples

### CODE REVIEW

**Comment by mmmries** [comment]
The workflows are fairly readable, but I would suggest that we not put them in the `lib` directory. Eveyrthing in the `lib` directory will be packaged up and published as part of the library. Since these workflows are examples of how to use the library, but not intended to be used in production codebase I think we should put them in an `examples` directory and then in the README we could have instructions on how to run something like `mix run examples/all_endpoints.exs`.

Making them `.exs` files also acts a hint that these are not intended to be compiled and loaded into another application. They are scripts that can be run once to show how the code can be used.

File: lib/atrium.ex [hunk 1]
```diff
@@ -1,41 +1,61 @@
 defmodule Atrium do
-  @mxAPIKEY "YOUR_MX_API_KEY"
-  @mxCLIENTID "YOUR_MX_CLIENT_ID"
-  @environment "ENVIRONMENT"
+  @mxAPIKEY "2f86c113535c59e19ccf5022e2fdfc284541fdba"
```
Comment (mmmries) [inline lib/atrium.ex]: I don't think you meant to add an API key to this file?

File: examples/accounts_and_transactions.exs [hunk 2]
```diff
@@ -0,0 +1,47 @@
+defmodule AccountsAndTransactions do
+  use Application
```
Comment (mmmries) [inline examples/accounts_and_transactions.exs]: `.exs` files will just execute top-to-bottom, so I think you can remove the `defmodule` here and the `def start(_type, _args)`. Then just start with the `IO.puts` as the beginning of the script

**Comment by brettmortensen** [comment]
Thank you @mmmries for all the help!

**Pull request merged**